lib/Driver/WindowsToolChains.cpp

```diff
@@ -76,6 +76,20 @@ toolchains::Windows::constructInvocation(const DynamicLinkJobAction &job,
   if (const Arg *A = context.Args.getLastArg(options::OPT_use_ld)) {
     Linker = A->getValue();
   }
+
+  if (Linker.empty() && context.OI.LTOVariant != OutputInfo::LTOKind::None)
+    Arguments.push_back("-fuse-ld=lld");
```

Comment:
Why don't we just set `Linker` here and then let the code after the `switch` add the flag?